CONTRIBUTING.md

```diff
@@ -0,0 +1,151 @@
+# Contributing to ubuntu-image
+
+This document covers the following topics:
+
+* [Setting up](#setting-up)
+* [Building](#building)
+* [Testing](#testing)
+* [Release process](#release-process)
+
+
+## Setting up
+
+Ubuntu 18.04 LTS or later is recommended for ubuntu-image development.
+Usually, the [latest LTS](https://releases.ubuntu.com/) would be the best choice.
+
+Go 1.21 [(or later)](https://go.dev/dl/) is required to build ubuntu-image.
```

Comment:
Since the beginning of the documentation is focused on ubuntu, should we point to the `go` snap here?

Reply:
Updated!